test/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -177,7 +177,7 @@ class RedirectBindingVerifier(private val response: IdpRedirectResponseDecorator
      */
     private fun verifyNoXMLSig() {
         if (response.responseDom.children("Signature").isNotEmpty()) {
-            throw SAMLComplianceException.create(SAMLBindings_3_4_4_1, message = "Signature element found.")
+            throw SAMLComplianceException.create(SAMLBindings_3_4_4_1, message = "Signature element found.", node = response.responseDom)
```

Comment:
❓ Is this line too long? Just making sure because it looks Strings are cut off early, like this one https://github.com/connexta/saml-conformance/pull/48/files#diff-9b82dc48c358096a39643f5081ef7f97R255

Reply:
Build passes, `gradle spotlessApply` doesn't seem to mind it

Reply:
https://github.com/connexta/saml-conformance/pull/49 this should resolve any line length issues